test/test_markup.rb

```diff
@@ -666,7 +666,7 @@
     index.add("greek/Bilbo-Baggins.md", "a [[Alpha|alpha.jpg]] b")
     index.commit("Add alpha.jpg")
 
-    page   = @wiki.page("Bilbo-Baggins")
+    page   = @wiki.page("greek/Bilbo-Baggins")
```

Comment:
These are relative paths; should I read them as being relative to root though?

Reply:
Yes, `./greek` in this case is the page file dir. Is that intuitive?

Reply:
> Yes, ./greek in this case is the page file dir. Is that intuitive?

Not to me. 😬 If `page_file_dir` is a system-wide setting set to `greek`, shouldn't it be assumed that when you call `@wiki.page("Bilbo-Baggins")` that you want `greek/Bilbo-Baggins`?

Reply:
Excuse me, you're right of course. `'greek'` in this example is not the page file dir. It is just a relative path that you should, indeed, read as relative to root. See below.

Reply:
OK.